monitoring/monitoring.go

```diff
@@ -388,60 +389,66 @@ func (monitor *Monitor) getCurrentSystemData() {
+	monitor.globalMonitoringData.UsedDisk, err = getSystemDiskUsage(monitor.workingDir)
 	if err != nil {
 		log.Errorf("Can't get system Disk usage: %s", err)
 	}
 
-	monitor.currentSystemData.InTraffic, monitor.currentSystemData.OutTraffic, err = monitor.trafficMonitoring.GetSystemTraffic()
+	inTraffic, outTraffic, err := monitor.trafficMonitoring.GetSystemTraffic()
 	if err != nil {
 		log.Errorf("Can't get system traffic value: %s", err)
 	}
 
+	monitor.globalMonitoringData.InTraffic = inTraffic
+	monitor.globalMonitoringData.OutTraffic = outTraffic
+
 	log.WithFields(log.Fields{
-		"CPU":  monitor.currentSystemData.Cpu,
-		"RAM":  monitor.currentSystemData.Ram,
-		"Disk": monitor.currentSystemData.UsedDisk,
-		"IN":   monitor.currentSystemData.InTraffic,
-		"OUT":  monitor.currentSystemData.OutTraffic,
+		"CPU":  monitor.globalMonitoringData.CPU,
+		"RAM":  monitor.globalMonitoringData.RAM,
+		"Disk": monitor.globalMonitoringData.UsedDisk,
+		"IN":   monitor.globalMonitoringData.InTraffic,
+		"OUT":  monitor.globalMonitoringData.OutTraffic,
 	}).Debug("Monitoring data")
 }
 
-func (monitor *Monitor) getCurrentServicesData() {
-	for serviceID, value := range monitor.serviceMap {
-		cpuUsage, err := getServiceCPUUsage(int32(value.uid))
+func (monitor *Monitor) getCurrentInstanceData() {
+	for instanceID, value := range monitor.instanceMonitoringMap {
+		cpuUsage, err := getInstanceCPUUsage(int32(value.uid))
 		if err != nil {
 			log.Errorf("Can't get service CPU: %s", err)
 		}
 
-		value.monitoringData.Cpu = uint64(math.Round(cpuUsage / float64(runtime.NumCPU())))
+		value.monitoringData.CPU = uint64(math.Round(cpuUsage / float64(runtime.NumCPU())))
 
-		value.monitoringData.Ram, err = getServiceRAMUsage(int32(value.uid))
+		value.monitoringData.RAM, err = getInstanceRAMUsage(int32(value.uid))
 		if err != nil {
 			log.Errorf("Can't get service RAM: %s", err)
 		}
 
-		value.monitoringData.UsedDisk, err = getServiceDiskUsage(monitor.storageDir, value.uid, value.gid)
+		value.monitoringData.UsedDisk, err = getInstanceDiskUsage(monitor.storageDir, value.uid, value.gid)
 		if err != nil {
 			log.Errorf("Can't get service Disc usage: %s", err)
 		}
 
-		value.monitoringData.InTraffic, value.monitoringData.OutTraffic, err = monitor.trafficMonitoring.GetServiceTraffic(serviceID)
+		inTraffic, outTraffic, err := monitor.trafficMonitoring.GetInstanceTraffic(instanceID)
 		if err != nil {
 			log.Errorf("Can't get service traffic: %s", err)
 		}
 
+		value.monitoringData.InTraffic = inTraffic
+		value.monitoringData.OutTraffic = outTraffic
+
 		log.WithFields(log.Fields{
-			"id":   serviceID,
-			"CPU":  value.monitoringData.Cpu,
-			"RAM":  value.monitoringData.Ram,
+			"id":   instanceID,
+			"CPU":  value.monitoringData.CPU,
+			"RAM":  value.monitoringData.RAM,
 			"Disk": value.monitoringData.UsedDisk,
 			"IN":   value.monitoringData.InTraffic,
 			"OUT":  value.monitoringData.OutTraffic,
```

Comment:
Change message to `Instance monitoring data`.

Reply:
Done

Reply:
Done